src/landscape/components/LandscapeForm/KeyInfoStep.js

```diff
@@ -38,7 +38,7 @@ const VALIDATION_SCHEMA = yup
   .object({
     name: yup.string().trim().required(),
     description: yup.string().max(MAX_DESCRIPTION_LENGTH).trim().required(),
-    location: yup.string().trim().required(),
+    location: yup.string().trim().selected().required(),
```

Comment:
Question: Do we need selected() and required() too? it seems that selected is a customized required check